spec/features/index_page_spec.rb

```diff
@@ -22,6 +22,15 @@
     expect(page).to have_content(customer.email)
   end
 
+  it "links to the customer show page without javascript", js: false do
```

Comment:
I think the default driver is currently RackTest for most of the feature specs.

11 / 56 feature specs enable javascript with the `:js` flag:

```
bundle exec rspec spec/features
# 56 examples, 0 failures

bundle exec rspec spec/features --tag js
# 11 examples, 0 failures
```

We could convert these to use the hyperlink, but it's quite handy to have the coverage & confidence the `data-url` / JS option still works.

What do you think @nickcharlton?